### DISCUSSION TITLE
Payment unconfirmed after 5 days(outgoing from btcpay server wallet)

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#5190] [Technical Support]
I did a test to see how btcpay server works. I created some transactions and paid them. The bitcoin ended up in my btcpay server wallet. 

I than sent them out of my btcpay server wallet(outgong). I sent the transaction 5 days ago and it's still unconfirmed. 

I picked the default transaction fee. I tried to bump the transaction fee but i won't work.

Inside btcpay server I clicked on "Actions" -> "Bump Fee(CPFP) but i get "There isn't any UTXO available to bump fee"

![noUTX](https://github.com/btcpayserver/btcpayserver/assets/31393558/1eaeb4d7-65e3-409f-a01b-efe8a7ec98c4)

![uncofimed](https://github.com/btcpayserver/btcpayserver/assets/31393558/f8ce1a7e-b72b-4d93-bfdd-a3bdf4f59ae4)

What could be wrong?

**Comment by Delitants** [comment]
Same problem here, I have selected a low fee initially and RBF got randomized to be off. Now this bump option gives me "There isn't any UTXO available to bump fee".
What is that suppose to mean? Why no answers here?

**Reply by ghost** [reply]
I got answers on their support chat.
https://chat.btcpayserver.org/btcpayserver/channels/support

They told me to do 2 things but it was way too much work and difficult. The problem was we set the fee too low and most miners don't want to take on the transaction. 

I was going to try what they told me but i was lazy and when i got around to trying it they transaction went through. 
It took 1 month for my transaction to go through.

**Comment by Delitants** [comment]
> I got answers on their support chat. https://chat.btcpayserver.org/btcpayserver/channels/support
> 
> They told me to do 2 things but it was way too much work and difficult. The problem was we set the fee too low and most miners don't want to take on the transaction.
> 
> I was going to try what they told me but i was lazy and when i got around to trying it they transaction went through. It took 1 month for my transaction to go through.

To do what??

**Reply by ghost** [reply]
Here you go. Don't ask me what they mean because i can't really explain it. 

1) CPFP if you control the receiving address (i.e. not if it's an exchange)

2) RBF in all other cases (not supported by the btcpay server wallet, you will need to connect your hw wallet or enter your seed in a desktop wallet like sparrow wallet) to perform an RBF

I actually made another transaction on purpose with a very low fee.  I wanted to see how long this one would take.

It's been stuck for 2 weeks now. I'm going to wait 2 months and see.

**Reply by Delitants** [reply]
So you just waited out or paid extra with CPFP/RBF? I can't say that my fee was that low, but suggested fee to pay $270 out of $1,900 of my transfer is a rip-off. I lowered it down to $80 something and it's now stuck for 3 days.

**Reply by ghost** [reply]
I waited out because i could not understand what RBF meant. I read into it a bit and what i understood was you download sparrow wallet and put in the btcpayserver seed. You would see your balance and you would send the transaction again with a higher fee and the old transaction would get cancelled. 
I never actually tried it because i felt it was a lot of work and i wanted to wait longer to see if it eventually went through. I was more worried that it would never go through. 
My amount was like 200USD and a fee of 1.67USD

The test transaction i am testing is 20.00USD and 0.67 cents

80 dollars for a transaction fee is still a rip off. I've never paid more then 5USD for a transaction. 
I refuse to pay more then 5USD for a transaction. If it takes a month i will wait. 
It's unfortunate miners are ripping us off. Business can't function on transactions that high. I use Bitcoin as long term savings so i send and keep it there for years.

**Comment by Delitants** [comment]
> Here you go. Don't ask me what they mean because i can't really explain it.
> 
> 1. CPFP if you control the receiving address (i.e. not if it's an exchange)
> 2. RBF in all other cases (not supported by the btcpay server wallet, you will need to connect your hw wallet or enter your seed in a desktop wallet like sparrow wallet) to perform an RBF

Great, CPFP not an option because I don't control receiving address. And RBF was not enabled during transaction (because BTCPayserver has it set to random choice by default).

**Reply by ghost** [reply]
![screen](https://github.com/btcpayserver/btcpayserver/assets/31393558/cd441d7f-47e3-4367-b944-b00328bd90b5)

So what you are saying is RBF is not possible? I use the demo btcpay server to do my transactions. I noticed "allow fee" says Randomize like you said.

For my test transaction i set it at yes. If it doesn't go though in 2 months i will try the RBF solution

**Comment by Delitants** [comment]
<img width="1014" alt="Screenshot 2023-12-06 at 5 09 26 PM" src="https://github.com/btcpayserver/btcpayserver/assets/2836790/d01ee56d-1f8d-4486-8e40-794a2feb7170">

**Reply by ghost** [reply]
Is it possible that the transaction will never ever go through and be stuck forever?
That's what i was worried about. No one really answered that for me.

For 80 fee i would except it to go through in 1 hour. It's a shame this is happened. Is it the btcpayserver? or what?
I used the android bitcoin wallet and another transaction too 7 days.
This has only happened to me 2 times. Rest of the time it takes 24 hours

im cheap and refused to pay high fees. It's greed

**Reply by dennisreimann** [reply]
> Is it possible that the transaction will never ever go through and be stuck forever?

Nope, even if it is purged from the mempools, the node should rebroadcast it.

**Comment by Delitants** [comment]
> So what you are saying is RBF is not possible? I use the demo btcpay server to do my transactions. I noticed "allow fee" says Randomize like you said.

As I have read, that default setting of randomization randomly select either RBF ON or OFF for that particular transaction for some sort of privacy (doesn't make much sense how does it make it private). And I left it default and my transaction got created without RBF enabled in it.

> Is it possible that the transaction will never ever go through and be stuck forever?

According to google it's very possible, but it should be dropped out of the pool and funds should return back to your wallet. But how long that will take is undefined, from weeks to months...

> For 80 fee i would except it to go through in 1 hour. It's a shame this is happened. Is it the btcpayserver? or what?

I think it depends on amount you send. AS I have mentioned, suggested by system fee was around $250 for $1,900 I tried to send. That's crazy. So I have moved slider down to $80 and it said that it will take one day, so I was ok with that. That 1 day estimate turned out to be misleading. Now 3 days and nothing. Tried this accelerator for free https://bitaccelerate.com/ but no effect whatsoever.
